### PR TITLE
Add a page on publicly available Radius artifacts

### PR DESCRIPTION
**Pull request by Reshrahim**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.radapp.io/community/contributing/docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### Summary
📰📝🎤

This pull request adds a new section and a new page to the community documentation of `radius-project/docs`, showcasing the various artifacts on Radius, such as press, blog posts, and presentations. The purpose is to increase the visibility and awareness of the Radius project and its features, benefits, and impact.

> _`Radius` project_
> _New docs page and overview_
> _Showcase in autumn_

### Walkthrough
*  Add a new page for artifacts on Radius ([link](https://github.com/radius-project/docs/pull/947/files?diff=unified&w=0#diff-996ad68831fc39b871d7fed8424d79d262031223fd2a95f80a45674fd27ed4dbR1-R29))
* Link to the new page from the community overview page ([link](https://github.com/radius-project/docs/pull/947/files?diff=unified&w=0#diff-e3770355c61f4e617f6541a7674d1d265d15868ffec8a8a852324e86b5716fc7R28-R31))

### CODE REVIEW

File: docs/content/community/overview.md [hunk 1]
```diff
@@ -25,3 +25,7 @@ If you would like to file Issues, access the source code, or use Codespaces plea
 ## How can I get involved?
 
 One of the easiest ways to contribute is to participate in discussions at community engagements or via the Discord server referenced above. We are always seeking feedback, especially for suggestions around improving the product and docs.
+
+## Artifacts on Radius
```
Comment (kachawla) [inline docs/content/community/overview.md]: Have we considered adding a top level tab, something like `Learn More` on https://radapp.io/, with a drop down menu for each of the artifact type? Putting it under `community` might make it hard to discover, and it feels like these resources should be quickly accessible when a new user visits the website.

File: docs/content/community/radius-artifacts.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- Ongoing posts at [Radius Blog](https://blog.radapp.io)
+
+## Press
+
+- [ Microsoft launches Radius, an open-source application platform for the cloud-native era - Techcrunch Oct 18,2023](https://techcrunch.com/2023/10/18/microsoft-launches-radius-an-open-source-application-platform-for-the-cloud/) 
+- [Microsoft Reimagines Modern Application Deployment With Radius Platform Forbes Oct 23,2023](https://www.forbes.com/sites/janakirammsv/2023/10/23/microsoft-reimagines-modern-application-deployment-with-radius-platform/?sh=6a78a37c76d7)
+- [Microsoft’s Radius and the future of cloud-native development - InfoWorld Oct 26,2023](https://www.infoworld.com/article/3709448/microsofts-radius-and-the-future-of-cloud-native-development.html)
+- [Microsoft Azure introduces Radius, an open-source development platform for multi-cloud computing - ZDNET Oct 27,2023](https://www.zdnet.com/article/microsoft-azure-introduces-radius-an-open-source-development-platform-for-multi-cloud-computing/)
+
+## Presentations
```
Comment (kachawla) [inline docs/content/community/radius-artifacts.md]: Just a suggestion, more commonly used for this type of content

```suggestion
## Videos
```

File: docs/content/community/overview.md (hunk 1 above)
Comment (Reshrahim) [reply docs/content/community/overview.md]: Great idea! I can link the same page as a tab from radapp.io. As we grow, we can split them by artifact type.

Comment (AaronCrawfis) [inline docs/content/community/overview.md]:
```suggestion
## Press and media coverage
```

Let's use a different term to describe the page. Artifacts is a little vague.

File: docs/content/community/overview.md [hunk 3]
```diff
@@ -25,3 +25,7 @@ If you would like to file Issues, access the source code, or use Codespaces plea
...
+
+Visit the [Artifacts on Radius]({{< ref radius-artifacts >}}) page for a list of press, blog posts, presentations, and other publicly available artifacts on Radius.
```
Comment (AaronCrawfis) [inline docs/content/community/overview.md]:
```suggestion
Visit the [press and media coverage docs]({{< ref media-coverage >}}) for a list of press articles, blog posts, presentations, and other coverage of Radius.
```

File: docs/content/community/radius-artifacts.md
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md:1]: Let's name this page `media-coverage.md`

File: docs/content/community/radius-artifacts.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "Artifacts on Radius"
+linkTitle: "Artifacts on Radius"
+description: "Information about press, blog posts, presentation and other artifacts on Radius"
+weight: 400
```
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md]:
```suggestion
type: docs
title: "Radius press coverage and media links"
linkTitle: "Press/media coverage"
description: "Learn about Radius through press articles, blog posts, presentation and other coverage of Radius"
weight: 400
```

File: docs/content/community/radius-artifacts.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Microsoft Blog posts
```
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md]: Let's make this more generic. We don't want to make it just Microsoft-specific

```suggestion
## Blog posts
```

File: docs/content/community/radius-artifacts.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+- [Radius Launch Oct 2023 - The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/) 
+- [Microsoft Open Source Blog Oct 2023 - Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)
+- Ongoing posts at [Radius Blog](https://blog.radapp.io)
```
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md]: Let's make this a markdown table instead. Something like:

| Title | Source | Date |
|-------|--------|------|
| [The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)] | Microsoft Azure Blog | Oct 16, 2023 |

File: docs/content/community/radius-artifacts.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+## Press
```
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md]: Let's 
```suggestion
## Press articles
```

File: docs/content/community/radius-artifacts.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+- [ Microsoft launches Radius, an open-source application platform for the cloud-native era - Techcrunch Oct 18,2023](https://techcrunch.com/2023/10/18/microsoft-launches-radius-an-open-source-application-platform-for-the-cloud/)
```
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md]: Let's make this a table as well

File: docs/content/community/radius-artifacts.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+- Ongoing posts at [Radius Blog](https://blog.radapp.io)
+
+## Press
+
+- [ Microsoft launches Radius, an open-source application platform for the cloud-native era - Techcrunch Oct 18,2023](https://techcrunch.com/2023/10/18/microsoft-launches-radius-an-open-source-application-platform-for-the-cloud/) 
+- [Microsoft Reimagines Modern Application Deployment With Radius Platform Forbes Oct 23,2023](https://www.forbes.com/sites/janakirammsv/2023/10/23/microsoft-reimagines-modern-application-deployment-with-radius-platform/?sh=6a78a37c76d7)
+- [Microsoft’s Radius and the future of cloud-native development - InfoWorld Oct 26,2023](https://www.infoworld.com/article/3709448/microsofts-radius-and-the-future-of-cloud-native-development.html)
+- [Microsoft Azure introduces Radius, an open-source development platform for multi-cloud computing - ZDNET Oct 27,2023](https://www.zdnet.com/article/microsoft-azure-introduces-radius-an-open-source-development-platform-for-multi-cloud-computing/)
+
+## Videos
```
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md]: Let's move this to the second slot, as the articles page is going to be super long over time and we want the shorter videos section to stand out.

File: docs/content/community/radius-artifacts.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+- [Mark Russinovich at the Linux Foundation Member Summit](https://aka.ms/radius-lfms)
+- [Introducing Radius - Open at Microsoft ](https://www.youtube.com/watch?v=mT_NWFnYn0A)
+
+## Cloud Native Compute Foundation Submission
+
+- [Radius Sandbox submission to CNCF](https://github.com/cncf/sandbox/issues/65)
```
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md]: This link probably doesn't make sense here. I'd remove it

File: docs/content/community/radius-artifacts.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+## Press
+
+- [ Microsoft launches Radius, an open-source application platform for the cloud-native era - Techcrunch Oct 18,2023](https://techcrunch.com/2023/10/18/microsoft-launches-radius-an-open-source-application-platform-for-the-cloud/) 
+- [Microsoft Reimagines Modern Application Deployment With Radius Platform Forbes Oct 23,2023](https://www.forbes.com/sites/janakirammsv/2023/10/23/microsoft-reimagines-modern-application-deployment-with-radius-platform/?sh=6a78a37c76d7)
+- [Microsoft’s Radius and the future of cloud-native development - InfoWorld Oct 26,2023](https://www.infoworld.com/article/3709448/microsofts-radius-and-the-future-of-cloud-native-development.html)
+- [Microsoft Azure introduces Radius, an open-source development platform for multi-cloud computing - ZDNET Oct 27,2023](https://www.zdnet.com/article/microsoft-azure-introduces-radius-an-open-source-development-platform-for-multi-cloud-computing/)
+
+## Videos
+
+- [Mark Russinovich at the Linux Foundation Member Summit](https://aka.ms/radius-lfms)
```
Comment (AaronCrawfis) [inline docs/content/community/radius-artifacts.md]: Let's make this a table as well

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Love this page!! Couple formatting and naming suggestions

File: docs/content/community/radius-artifacts.md (hunk 5 above)
Comment (Reshrahim) [reply docs/content/community/radius-artifacts.md]: good catch!

File: docs/content/community/_index.md [hunk 12]
```diff
@@ -3,5 +3,5 @@ type: docs
 title: "Radius Community"
 linkTitle: "Community"
 description: "Information about the Radius community"
-weight: 90
+weight: 300
```
Comment (Reshrahim) [inline docs/content/community/_index.md]:
```suggestion
weight: 90
```

File: docs/content/community/media-coverage.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+linkTitle: "Press/media coverage"
+description: "Learn about Radius through press articles, blog posts, presentation and other coverage of Radius"
+weight: 400
+---
+
+## Blog posts
+
+| Blog Title | Source | Date |
+| ---------- | ------ | ---- |
+|[Radius Launch Oct 2023 - The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)|Microsoft Azure Blog|Oct 18,2023|
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md]:
```suggestion
|[The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)|Microsoft Azure Blog|Oct 18,2023|
```

File: docs/content/community/media-coverage.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+|[Microsoft Open Source Blog Oct 2023 - Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)|Microsoft Open Source Blog|Oct 18,2023|
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md]:
```suggestion
|[Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)|Microsoft Open Source Blog|Oct 18,2023|
```

File: docs/content/community/media-coverage.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+Ongoing blog posts can be found at [Radius Blog](https://blog.radapp.io)
+
+## Videos
+
+| Video | Event | Date |
+| ----- | ----- | ---- |
+|[Mark Russinovich at the Linux Foundation Member Summit](https://aka.ms/radius-lfms)|Linux Foundation Member Summit | Oct 24,2023 |
+|[Introducing Radius - Open at Microsoft ](https://www.youtube.com/watch?v=mT_NWFnYn0A)| Open at Microsoft | Nov 7,2023 |
+
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md:27]:
```suggestion
| [Radius, a new open-source application platform for the cloud](https://www.youtube.com/watch?v=gaG77PiYv5w) | Microsoft Ignite | Nov 17, 2023 |

```

Comment (Reshrahim) [reply docs/content/community/media-coverage.md:27]: just added this :P the timing!

File: docs/content/community/media-coverage.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+| Blog Title | Source | Date |
+| ---------- | ------ | ---- |
+|[Radius Launch Oct 2023 - The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)|Microsoft Azure Blog|Oct 18,2023|
+|[Microsoft Open Source Blog Oct 2023 - Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)|Microsoft Open Source Blog|Oct 18,2023|
+
+Ongoing blog posts can be found at [Radius Blog](https://blog.radapp.io)
+
+## Videos
+
+| Video | Event | Date |
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md:20]: For the dates can we consistently have the same sorting? My preference would be newest on top with oldest on bottom.

Comment (Reshrahim) [reply docs/content/community/media-coverage.md:20]: yes updated this in the last commit

File: docs/content/community/media-coverage.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+linkTitle: "Press/media coverage"
+description: "Learn about Radius through press articles, blog posts, presentation and other coverage of Radius"
+weight: 400
+---
+
+## Blog posts
+
+| Blog Title | Source | Date |
+| ---------- | ------ | ---- |
+|[The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)|Microsoft Azure Blog|Oct 18,2023|
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md]:
```suggestion
|[The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)|Microsoft Azure Blog|Oct 18, 2023|
```

File: docs/content/community/media-coverage.md [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+|[Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)|Microsoft Open Source Blog|Oct 18,2023|
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md]:
```suggestion
|[Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)|Microsoft Open Source Blog|Oct 18, 2023|
```

File: docs/content/community/media-coverage.md [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+
+Ongoing blog posts can be found at [Radius Blog](https://blog.radapp.io)
+
+## Videos
+
+| Video | Event | Date |
+| ----- | ----- | ---- |
+|[ Brendan Burns and Mark Russinovich at Ignite about Radius](https://www.youtube.com/watch?v=gaG77PiYv5w&ab_channel=MicrosoftIgnite)| Microsoft Ignite | Nov 17,2023 |
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md]:
```suggestion
|[ Brendan Burns and Mark Russinovich at Ignite about Radius](https://www.youtube.com/watch?v=gaG77PiYv5w&ab_channel=MicrosoftIgnite)| Microsoft Ignite | Nov 17, 2023 |
```

File: docs/content/community/media-coverage.md [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+|[Introducing Radius - Open at Microsoft ](https://www.youtube.com/watch?v=mT_NWFnYn0A)| Open at Microsoft | Nov 7,2023 |
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md]:
```suggestion
|[Introducing Radius - Open at Microsoft ](https://www.youtube.com/watch?v=mT_NWFnYn0A)| Open at Microsoft | Nov 7, 2023 |
```

File: docs/content/community/media-coverage.md [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+|[Mark Russinovich at the Linux Foundation Member Summit](https://aka.ms/radius-lfms)|Linux Foundation Member Summit | Oct 24,2023 |
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md]:
```suggestion
|[Mark Russinovich at the Linux Foundation Member Summit](https://aka.ms/radius-lfms)|Linux Foundation Member Summit | Oct 24, 2023 |
```

File: docs/content/community/media-coverage.md [hunk 22]
```diff
@@ -0,0 +1,35 @@
...
+
+
+## Press articles
+
+| Article Title | Media Source | Date |
+| ------------- | ------------ | ---- |
+|[ Microsoft launches Radius, an open-source application platform for the cloud-native era](https://techcrunch.com/2023/10/18/microsoft-launches-radius-an-open-source-application-platform-for-the-cloud/)| TechCrunch | Oct 18,2023 |
+|[Microsoft reimagines Modern Application Deployment With Radius Platform](https://www.forbes.com/sites/janakirammsv/2023/10/23/microsoft-reimagines-modern-application-deployment-with-radius-platform/?sh=6a78a37c76d7)| Forbes | Oct 23,2023 |
+|[Microsoft’s Radius and the future of cloud-native development](https://www.infoworld.com/article/3709448/microsofts-radius-and-the-future-of-cloud-native-development.html)| InfoWorld | Oct 26,2023 |
+|[Microsoft Azure introduces Radius, an open-source development platform for multi-cloud computing](https://www.zdnet.com/article/microsoft-azure-introduces-radius-an-open-source-development-platform-for-multi-cloud-computing/)| ZDNET | Oct 27,2023 |
```
Comment (AaronCrawfis) [inline docs/content/community/media-coverage.md]:
```suggestion
|[Microsoft Azure introduces Radius, an open-source development platform for multi-cloud computing](https://www.zdnet.com/article/microsoft-azure-introduces-radius-an-open-source-development-platform-for-multi-cloud-computing/)| ZDNET | Oct 27, 2023 |
|[Microsoft’s Radius and the future of cloud-native development](https://www.infoworld.com/article/3709448/microsofts-radius-and-the-future-of-cloud-native-development.html)| InfoWorld | Oct 26, 2023 |
|[Microsoft reimagines Modern Application Deployment With Radius Platform](https://www.forbes.com/sites/janakirammsv/2023/10/23/microsoft-reimagines-modern-application-deployment-with-radius-platform/?sh=6a78a37c76d7)| Forbes | Oct 23, 2023 |
|[ Microsoft launches Radius, an open-source application platform for the cloud-native era](https://techcrunch.com/2023/10/18/microsoft-launches-radius-an-open-source-application-platform-for-the-cloud/)| TechCrunch | Oct 18, 2023 |
```

**Review by AaronCrawfis — APPROVED** [review]
Overall looks good. Just some small suggestions on date format and reordering the articles section. Thanks!

**Pull request merged**